Evaluation/Input/evaluation_plan.json

```diff
@@ -180,85 +183,73 @@
             "Project": "Compound",
             "Name": "COMPOUND",
             "Type": "Compound",
-            "SectionId": 31
+            "SectionId": 0
```

Comment:
here I kept the SectionID instead of SectionReference because I just found a bug :)
https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/issues/899 
Once it was fixed, I will replace it here as well.